src/test/java/eu/dissco/core/datacitepublisher/TestUtils.java

```diff
@@ -0,0 +1,220 @@
+package eu.dissco.core.datacitepublisher;
+
+import static eu.dissco.core.datacitepublisher.schemas.DigitalSpecimen.MaterialSampleType.ORGANISM_PART;
+
+import com.fasterxml.jackson.annotation.JsonInclude.Include;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.module.SimpleModule;
+import com.fasterxml.jackson.dataformat.xml.XmlMapper;
+import eu.dissco.core.datacitepublisher.configuration.InstantDeserializer;
+import eu.dissco.core.datacitepublisher.configuration.InstantSerializer;
+import eu.dissco.core.datacitepublisher.domain.datacite.DcAlternateIdentifier;
+import eu.dissco.core.datacitepublisher.domain.datacite.DcAttributes;
+import eu.dissco.core.datacitepublisher.domain.datacite.DcContributor;
+import eu.dissco.core.datacitepublisher.domain.datacite.DcCreator;
+import eu.dissco.core.datacitepublisher.domain.datacite.DcDate;
+import eu.dissco.core.datacitepublisher.domain.datacite.DcDescription;
+import eu.dissco.core.datacitepublisher.domain.datacite.DcNameIdentifiers;
+import eu.dissco.core.datacitepublisher.domain.datacite.DcRelatedIdentifiers;
+import eu.dissco.core.datacitepublisher.domain.datacite.DcSubject;
+import eu.dissco.core.datacitepublisher.domain.datacite.DcTitle;
+import eu.dissco.core.datacitepublisher.domain.datacite.DcType;
+import eu.dissco.core.datacitepublisher.domain.datacite.UriScheme;
+import eu.dissco.core.datacitepublisher.schemas.DigitalSpecimen;
+import eu.dissco.core.datacitepublisher.schemas.DigitalSpecimen.MaterialSampleType;
+import eu.dissco.core.datacitepublisher.schemas.DigitalSpecimen.TopicCategory;
+import eu.dissco.core.datacitepublisher.schemas.DigitalSpecimen.TopicDiscipline;
+import eu.dissco.core.datacitepublisher.schemas.DigitalSpecimen.TopicDomain;
+import eu.dissco.core.datacitepublisher.schemas.MediaObject;
+import eu.dissco.core.datacitepublisher.schemas.MediaObject.LinkedDigitalObjectType;
+import eu.dissco.core.datacitepublisher.schemas.MediaObject.MediaFormat;
+import java.time.Instant;
+import java.util.List;
+
+public class TestUtils {
+
+  private TestUtils() {
+  }
+
+  public static final String PID = "https://doi.org/10.3535/QR1-P21-9FW";
```

Comment:
Probably better to do `"https://doi.org/10.3535/ + SUFFIX` and the same for the DOI one.